packages/manager/apps/telecom/src/app/telecom/telephony/alias/dashboard/dashboard.controller.js

```diff
@@ -305,11 +303,8 @@ export default class TelecomTelephonyAliasHomeController {
   groupNumberByFeatureType() {
     switch (this.alias.featureType) {
       case 'contactCenterSolution':
-      case 'esayHunting':
-      case 'miniPabx':
-        return 'easyHunting';
+      case 'easyHunting':
```

Comment:
Are you sure "easyHunting" should return 'redirect' ?

Reply:
I've just corrected the typo
And it seems to me that easyHunting doesn't exist any more either, it's the contactCenterSolution that has replaced this feature.